src/test/java/uk/ac/ebi/eva/pipeline/jobs/GenotypedVcfJobWorkflowTest.java

```diff
@@ -240,7 +236,7 @@ public void setUp() throws Exception {
     }
 
     private void initVariantConfigurationJob() {
-        jobOptions.getPipelineOptions().put(JobParametersNames.DB_NAME, mongoRule.getRandomTemporaryDatabaseName());
+        mongoRule.getTemporaryDatabase(jobOptions.getDbName());
```

Comment:
Not yet, I checked with @jmmut yesterday and it can't be done without job parameters.